lnd_test.go

```diff
@@ -1025,6 +1028,7 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 	closeChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
 	ctxt, _ = context.WithTimeout(ctxb, timeout)
 	closeChannelAndAssert(ctxt, t, net, net.Bob, chanPoint2, false)
+	ctxt, _ = context.WithTimeout(ctxb, timeout)
```

Comment:
👍 Good catch!  The `ctxt, _ = context.WithTimeout` below can be removed, as it is obviously misplaced.

Reply:
Removed. Thanks for noticing